components/EventListItem/EventListItem.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div class="event-list-item">
+    <div class="event-content">
+      <div class="image">
+        <event-banner-image :src="bannerUrl" />
+      </div>
+      <div class="event-content-wrap">
+        <h3>
+          <a :href="event.fields.url" target="_blank">
+            {{ event.fields.title }}
+          </a>
+        </h3>
+        <div class="subtitle">
+          <div>{{ event.fields.location }}</div>
+          <!-- <div>{{ startDate }} - {{ endDate }}</div> -->
```

Comment:
Remove if not needed. I think I commented this out earlier on when I was getting errors that it was undefined.